lib/matplotlib/tests/test_mathtext.py

```diff
@@ -12,7 +12,7 @@
 from matplotlib import mathtext
 
 math_tests = [
-    r'$a+b+\dots+\dot{s}+\ldots$',
+    r'$a+b+\dot s+\dot{s}+\ldots$',
```

Comment:
Well, I just checked and it appears I was wrong. Sorry for that. ``\dots`` should indeed produce three dots and is a text and math command, whereas ``\ldots`` (and other ```cdots``, etc) are math-only commands. I checked it in (the LaTeX comprehensive symbol list)[https://www.ctan.org/tex-archive/info/symbols/comprehensive/].